Adding a separate Dockerfile for local openshift registry (#1189)
tl-its-umich-edu/my-learning-analytics: dockerfiles/README.md

> The Dockerfile(s) for the build are in this directory

Reviewer: ```suggestion
The Dockerfile(s) for the build are in this directory.
```

> * Dockerfile is the file used for most builds including localhost

Reviewer: ```suggestion
* `Dockerfile` is the file used for most builds including when using localhost.
```

> * Dockerfile.openshift is a custom file with the FROM entries modified to point
> to local images in the Openshift repository.

Reviewer: ```suggestion
* Dockerfile.openshift is a custom file with the FROM entries modified to point
  to local images in the Openshift repository.
  (Note: we may be able to combine this with `Dockerfile` again when we have access Openshift 4).
```

If you're using Openshift you'll want to put both python and node into 

> into the openshift image stream then configure your build YAML like:

Reviewer: ```suggestion
into the openshift image stream then configure your build YAML to use the following `strategy` block:
```

> ``

Reviewer: You're missing a a ` here. I couldn't get the suggestion to work.

Author: Thanks, fixed.

The Dockerfile(s) for the build are in this directory.

* `Dockerfile` is the file used for most builds including when using localhost.

* `Dockerfile.openshift` is a custom file with the FROM entries modified to point
  to local images in the Openshift repository.

>   (Note: we may be able to combine this with `Dockerfile` again when we have access Openshift 4).

Reviewer: ```suggestion
  (Note: we may be able to combine this with `Dockerfile` again when we have access to Openshift 4).
```

Author: Sorry, merged before I saw this one. Documentation is harder to write than code. ;)

Author: Hmm, is the period supposed to be in the parenthesis? This looks correct. . . But
["When a complete, independent sentence is entirely enclosed by parentheses, the period goes inside the closing parenthesis."](https://www.grammarly.com/blog/period/)

Didn't know about that edge case!

Reviewer: Yeah, I was fixing my own mistakes here. It makes sense to me from a grammar perspective. A period ends an independent clause, which in this case is entirely in the parentheses. Not a super important thing, but something I would definitely have wanted corrected when I was working as a journalist.
